### DISCUSSION TITLE
I want to run the flet app offline too

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#609] [Q&A]
My app is a simple calculator app for which user does not need internet so any way run flet app without internet

**Comment by FeodorFitsner** [comment]
You can run Flet app offline already. No internet connection required.

**Reply by plugeit** [reply]
https://todo.factfry.repl.co/#/
this is a example website so if i turn off the internet it says reconnecting....

**Comment by FeodorFitsner** [comment]
Oh, I see. Unfortunately, such scenario is not supported as Python code is running on a web server. So, Flet app can be installed as PWA to your desktop or mobile, but it's a "connected" app, i.e. app requiring a backend.
